docs/microbit_micropython_api.rst

```diff
@@ -105,6 +139,34 @@ Each of these pins are instances of the ``MicroBitPin`` class, which offers the
     # returns boolean
     pin.is_touched()
```

Comment:
We should probably indicate here that this is only for the touch pins 0,1,2.

Reply:
Done